.travis.yml

```diff
@@ -4,38 +4,128 @@ notifications:
   email: false
 language: node_js
 node_js:
-  - "10"
   - "8"
-  - "6"
-jdk:
-  - oraclejdk9
-  - openjdk8
-env:
-  - MAVEN_VERSION=3.6.1 JAVA_VERSION=openjdk8
-  - MAVEN_VERSION=3.6.1 JAVA_VERSION=oraclejdk9
 
-  - MAVEN_VERSION=3.3.9 JAVA_VERSION=oraclejdk9
-  - MAVEN_VERSION=3.3.9 JAVA_VERSION=openjdk8
+cache:
+  directories:
+    - node_modules
+    - $HOME/.m2
 
 before_install:
-  # sdkman helps to install different gradle versions
+  # sdkman helps to install different maven versions
   - curl -s "https://get.sdkman.io" | bash
   - source "$HOME/.sdkman/bin/sdkman-init.sh"
   - echo sdkman_auto_answer=true > ~/.sdkman/etc/config
-cache:
-  directories:
-    - node_modules
-script:
-  - jdk_switcher use $JAVA_VERSION
-  - sdk install maven $MAVEN_VERSION
-  - npm test
+  # install jdk is needed for JDK10+
+  - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
+
 jobs:
   include:
+    # test MAVEN_VERSION=3.6.1
```

Comment:
please drop any commented out code you don't need any more

Reply:
Done